bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -228,6 +228,10 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                std::chrono::seconds,
                std::chrono::seconds{3600},
                "Interval time to create db snapshot in seconds");
+  CONFIG_PARAM(dbCheckpointMonitorIntervalSeconds,
+               std::chrono::seconds,
+               std::chrono::seconds{60},
+               "Monitor time in seconds to removing oldest db checkpoint when disk space exhausted");
```

Comment:
Description: "Time interval in seconds to monitor disk usage by db checkpoints"?

Reply:
fixed